src/main/java/duke/AddDeadlineCommand.java

```diff
@@ -0,0 +1,22 @@
+package duke;
+
+import java.time.LocalDate;
+
+public class AddDeadlineCommand extends Command {
```

Comment:
Perhaps some javadocs could be written for the methods and fields such that anyone who reads it can understand the code changes instantly 😄